tf2_bullet/CMakeLists.txt

```
@@ -46,5 +46,4 @@ if(BUILD_TESTING)
 endif()
 
 ament_export_include_directories(include)
-ament_export_libraries(${PROJECT_NAME})
-ament_package()
+ament_package(CONFIG_EXTRAS_POST bullet-extra.cmake)
```

I think we can just use `CONFIG_EXTRAS` here:

```suggestion
ament_package(CONFIG_EXTRAS bullet-extra.cmake)
```

Also, as convention, we almost always use "extras" as opposed to "extra", so rename this file to `bullet-extras.cmake`.